recipes/weeder/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: weeder
+  version: "2.0"
+
+source:
+  fn: weeder2.0.tar.gz
+  url:  http://159.149.160.51/modtools/downloads/weeder2.0.tar.gz
```

Comment:
That does not look like a stable URL. Bioconda will backup the tarball for you, but you should try to provide a stable URL at first hand.